src/platform/test/functional/apps/dashboard/group6/dashboard_esql_chart.ts

```diff
@@ -41,7 +42,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await dashboard.navigateToApp();
       await dashboard.clickNewDashboard();
       await timePicker.setDefaultDataRange();
-      await dashboard.switchToEditMode();
```

Comment:
No need to switch to edit mode, as it is already in edit mode.
This just waste 20 seconds while checking the mode